CCPAConsentViewController/Classes/CCPAConsentViewController.swift

```diff
@@ -266,6 +266,7 @@ extension CCPAConsentViewController: ConsentDelegate {
     }
 
     public func onAction(_ action: Action, consents: PMConsents?) {
+        consentDelegate?.onAction?(action, consents: consents)
         if action == .AcceptAll || action == .RejectAll || action == .SaveAndExit {
```

Comment:
@andresilveirah  In GDPR we are taking care of Dismiss action also, what you think, here also we need to take care of Dismiss action explicitly.

Reply:
@vilasma I thought about that as well. Since in CCPA we don't have a `PMCancel` action, it won't be necessary to treat the `Dismiss` action differently for the `onAction` delegate method. 
We might have to do something regarding the `onConsentReady` but this is not in the scope of this PR. Here all we want to make sure is that the `onAction` callback is fired and our clients can use it in their apps.